src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final ArrayList<Task> LIST_OF_THINGS = new ArrayList<>();
```

Comment:
Maybe you could consider extracting this out as a separate TaskList class, which would handle adding tasks, etc.